README.md

```diff
@@ -39,23 +39,32 @@ help
 `plotsbot-ryzokuken help [<module>...]`: Prints out this descriptive help message for each mentioned module. If no modules are specified, print the help message for ALL modules.`chatbot` is not the name of a valid module. Try looking up the `chatbot` module instead.
 ```
 
-## Services
+## Behaviors
 
-Services are different from interfaces as they carry out computational tasks and not I/O tasks like interfaces do. Therefore, the bot takes input from an interface, directs it to a service which would preform actions on the data and return data which the bot will output through another interface.
+Behaviors are clearly defined tasks performed by the bot. A behavior may use any third-party functions to perform this action, but it needs to expose a have a few mandatory fields that are used by the chatbot to trigger the behavior at the appropriate time.
 
-### Chatbot
+**Behavior Specification**:
+1. `trigger: String` => The event on which the behavior is supposed to be triggered. Currently, two events are supported - the `join` event which is triggered whenever a new user joins a particular channel, and the `message` event whenever a particular user sends a message on a channel.
+2. `action: Function` => This is the action of the behavior which is called by the bot at the appropriate moment. The bot provides the function with different arguments depending on the corresponding behavior's `trigger` value. For behaviors with `trigger` equal to `join`, the bot passes four arguments to the action function upon trigger. These are:
+  1 `channel: String` => This is the name of the channel on which the user joined.
```

Comment:
@ananyo2012 we didn't need it for `greet`, but we might. Especially when someone new joins. If someone joins on matrix, they stay there. But if someone joins on IRC and disconnects and joins again, the message shows up again. Sebastian and I both rejoin frequently and that might spam the channel. Later, the greet behavior will have to store the usernames of people it has greeted in a database to avoid greeting people repeatedly on the same channel. What do you say? Should we remove the `channel` string for the time-being and add it later when we need it?